crates/egui/src/widgets/plot/items/mod.rs

```diff
@@ -1019,6 +1021,12 @@ impl Arrows {
         self
     }
 
+    /// Set the size of the arrow tips
+    pub fn tip_size(mut self, tip_size: f32) -> Self {
+        self.tip_size = Some(tip_size);
```

Comment:
"size" is not very clear. Is it with width or the length of the tip? Calling it `tip_length` would make more sense imho

Reply:
Ok, I'm fine with "length". I already changed the name.